bundles/org.openhab.binding.qolsysiq/README.md

```diff
@@ -0,0 +1,125 @@
+|-------------|--------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------|----------------------------|
+| armState    | String | RW         | Reports the current partition arm state or sends an arm or disarm command to the system. Security codes can be appended to the command using a colon delimiter (e.g. 'DISARM:123456'). Codes appended to the command will be used in place of the `armCode` configuration property if set. | ALARM, ARM_AWAY, ARM_STAY, DISARM, ENTRY_DELAY, EXIT_DELAY | ARM_AWAY, ARM_STAY, DISARM |
+| alarmState  | String | RW         | Reports on the current alarm state, or triggers an instant alarm                                                                                                                                                                                                                           | AUXILIARY, FIRE, POLICE, ZONEOPEN, NONE                    | AUXILIARY, FIRE, POLICE    |
+| armingDelay | Number | R          | The arming delay countdown currently in progress                                                                                                                                                                                                                                           | Seconds remaining                                          | N/A                        |
+| errorEvent  | String | R          | Last error event message reported by the partition. Clears after 30 seconds                                                                                                                                                                                                                | Error text                                                 | N/A                        |
+
+### Zone Channels
+
+| Channel | Type    | Read/Write | Description            | State Options                               | 
+|---------|---------|------------|------------------------|---------------------------------------------|
+| status  | String  | R          | The zone status        | ACTIVE, CLOSED, OPEN, FAILURE, IDLE, TAMPER |
+| state   | Number  | R          | The zone state         | Number                                      |
+| contact | Contact | R          | The zone contact state | OPEN, CLOSED                                |
+
+## Full Example
+
+### qolsysiq.things
+
+```
+Bridge qolsysiq:panel:home "Home Security Panel" [ hostname="192.168.3.123", port=12345, key="AAABBB00" ] {
+    Bridge partition 0 "Partition Main" [ id=0, armCode="123456" ] {
+        Thing zone 0 "Window" [ id=1 ]
+        Thing zone 1 "Motion" [ id=2 ]
```

Comment:
I thing thing ids should be 1 and 2 instead of 0  and 1 to be compliant with your channel links below.

Reply:
good catch ;-)